string altitude_string
---
bool success
